src/Features/LanguageServer/Protocol/Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj

```diff
@@ -21,6 +21,7 @@
     <Reference Include="System.ComponentModel.Composition" />
     <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol.Extensions" Version="$(MicrosoftVisualStudioLanguageServerProtocolExtensionsVersion)" />
     <PackageReference Include="StreamJsonRpc" Version="$(StreamJsonRpcVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
```

Comment:
what for?

Reply:
We are moving to a newer `NewtonsoftJsonVersion` in the PR and without this change I get a build warning of mismatched versions, which asks me to explicitly add a package reference in project to the desired version.

Reply:
Fix is correct. And FYI to @dpoeschl who was also fighting this I believe.